includes/discovery/sensors/temperature/cmts.inc.php

```diff
@@ -0,0 +1,20 @@
+<?php
+$oids = snmpwalk_cache_oid_num($device, ".1.3.6.1.4.1.4998.1.1.10.1.4.2.1.29");
+
+foreach ($oids as $index => $data) {
+    $tempCurr = implode("", $data);
+    if ($tempCurr !== "999") {
+        $split_oid        = explode('.', $index);
+        $temperature_id   = $split_oid[(count($split_oid) - 1)];
+        $temperature_oid  = ".1.3.6.1.4.1.4998.1.1.10.1.4.2.1.29.1.$temperature_id";
+        $descr_oid        = ".1.3.6.1.4.1.4998.1.1.10.1.4.2.1.3.1.$temperature_id";
+        $warnlimit_oid    = ".1.3.6.1.4.1.4998.1.1.10.1.4.2.1.44.1.$temperature_id";
+        $limit_oid        = ".1.3.6.1.4.1.4998.1.1.10.1.4.2.1.45.1.$temperature_id";
+
+        $descr            = trim(snmp_get($device, $descr_oid, '-Oqv', ''), '"');
```

Comment:
It might be better to walk all of these up front. For example:
```php
$oids = snmpwalk_cache_oid($device, 'cardTemperature', array(), 'CADANT-CMTS-EQUIPMENT-MIB');
$oids = snmpwalk_cache_oid($device, 'cardName', $oids, 'CADANT-CMTS-EQUIPMENT-MIB');
$oids = snmpwalk_cache_oid($device, 'cardTemperatureHighWarn', $oids, 'CADANT-CMTS-EQUIPMENT-MIB');
$oids = snmpwalk_cache_oid($device, 'cardTemperatureHighError', $oids, 'CADANT-CMTS-EQUIPMENT-MIB');

foreach ($oids as $index => $entry) {
$tempCurr = $entry['cardTemperature'];
    if ($tempCurr !== "999") {
        $temperature_oid  = ".1.3.6.1.4.1.4998.1.1.10.1.4.2.1.29.$index";
        $descr = $entry['cardName'];
        $warnlimit = $entry['cardTemperatureHighWarn'];
        $limit = $entry['cardTemperatureHighError'];

        // discover_sensor ...
    }
}


```